pkg/operator/ceph/pool/controller.go

```diff
@@ -434,9 +434,9 @@ func configureRBDStats(clusterContext *clusterd.Context, clusterInfo *cephclient
 	logger.Debugf("RBD per-image IO statistics will be collected for pools: %v", enableStatsForPools)
 	monStore := config.GetMonStore(clusterContext, clusterInfo)
 	if len(enableStatsForPools) == 0 {
-		err = monStore.Delete("mgr.", "mgr/prometheus/rbd_stats_pools")
+		err = monStore.Delete("mgr", "mgr/prometheus/rbd_stats_pools")
 	} else {
-		err = monStore.Set("mgr.", "mgr/prometheus/rbd_stats_pools", strings.Join(enableStatsForPools, ","))
+		err = monStore.Set("mgr", "mgr/prometheus/rbd_stats_pools", strings.Join(enableStatsForPools, ","))
```

Comment:
Should we also consider removing the old `mgr.` setting for upgrade?

Reply:
@travisn . Done , squashed to a single commit.

Reply:
@shalevpenker97 From your testing in #10574, I can't tell... For upgraded clusters, if this setting is set for both `mgr.` (previous behavior) and `mgr` (new behavior), is the issue resolved even if the `mgr.` is not deleted? Otherwise, we would need to delete the `mgr.` setting as Blaine suggested. But if the new setting works even with the old one still applied, let's just merge this PR as is.

Reply:
@travisn , I can verify that no need to remove `mgr.`.
Adding the `mgr` enabled the metrics.